src/asmjit/core/virtmem.cpp

```diff
@@ -581,22 +618,76 @@ static HardenedRuntimeFlags getHardenedRuntimeFlags() noexcept {
   return flags;
 }
 
-Error alloc(void** p, size_t size, MemoryFlags memoryFlags) noexcept {
-  *p = nullptr;
-  if (size == 0)
-    return DebugUtils::errored(kErrorInvalidArgument);
-
+static Error mMap(void** p, size_t size, MemoryFlags memoryFlags) noexcept {
```

Comment:
Nitpick: I would love some better name, `mMap()` is kinda unfortunate :)

Reply:
No problem.  Instead of the abbreviated `mMap` we expand the `m` out to `memoryMap`?

Reply:
Yeah that sounds good